services/src/api/handlers/layers.rs

```diff
@@ -1228,10 +1229,11 @@ mod tests {
     use tokio_postgres::NoTls;
 
     #[ge_context::test]
-    async fn test_add_layer_to_collection(app_ctx: PostgresContext<NoTls>) {
-        let ctx = app_ctx.default_session_context().await.unwrap();
+    async fn test_add_layer_to_collection(app_ctx: ProPostgresContext<NoTls>) {
+        let session = admin_login(&app_ctx).await;
```

Comment:
in most of the adp tests, the first thing we do is to either create an anonymous session or login as admin, and then the the session context. would it make sense to encapsulate this into the test macro? like ge_context::api_test, ge_context::admin_api_test, that does the login and context creation and passes them into the test function? Could be a separate task to do something like that ofc.

Reply:
Gute Idee, aber nicht in dem PR.